client/app/scripts/superdesk/editor/editor.js

```diff
@@ -1225,6 +1267,18 @@ angular.module('superdesk.editor', ['superdesk.editor.spellcheck', 'angular-embe
                 }
             }
         };
-    }]);
+    }]).config(['embedlyServiceProvider', 'config', function(embedlyServiceProvider, config) {
+        // embed.ly private key
+        console.log(config);
```

Comment:
development, it's still not working. This `config` doesn’t contain what it should at this point